addons/l10n_ec/data/account_fiscal_position_template.xml

```diff
@@ -1,55 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <record id="fp_companies" model="account.fiscal.position.template">
+    <record id="fp_local" model="account.fiscal.position.template">
+        <field name="sequence">10</field>
         <field name="chart_template_id" ref="l10n_ec_ifrs"/>
-        <field name="name">Sociedades - personas juridicas</field>
+        <field name="name">Régimen nacional</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_id" ref="base.ec"/> 
     </record>
-    <record id="fp_special_taxation_companies" model="account.fiscal.position.template">
+    <record id="fp_foreing" model="account.fiscal.position.template">
```

Comment:
@pepetreshere will map only for goods (Fiscal Position 0% for goods).
Then if the user needs to use services, they will duplicate this Fiscal Position and map the taxes.

Reply:
But then we can not apply it automatically.  Ideally we should split the domestic tax for goods and services then, but we might already have split a lot.

Reply:
@jco-odoo @vbe-odoo @pepetreshere let's discuss this point. It has been a long time since we discussed this, I am writing down my understanding of it so we all have the information and can get to an agreement:

For sales:
- For domestic 0%, it needs to be mapped either to "0% Exports VAT for Goods" or "0% Exports VAT for Services" (like mapping tax code 415 to tax code 417 or 418)
- For domestic 12%, it needs to be mapped either to "0% Exports VAT for Goods" or "0% Exports VAT for Services" (like mapping tax code 411 to tax code 417 or 418)
- There are other sale taxes such as "12% VAT for Assets", etc, those also could be mapped to the same "0% Exports VAT for Goods" or "0% Exports VAT for Services".
- It is also customary to change the domestic receivable account for the foreign receivable account

For purchases (this is being confirmed by Karina, let's wait a bit on this one):
- For goods, they come in **through customs**, keep note that most countries export free of VAT, anyway no VAT should be recorded in the invoice, if there were any it should be incorporated into the price of the product.
- For services, they come in **through a purchase liquidation**. The foreign VAT is incorporated into the price and on top of it the user should register the local VAT.
- It is also customary to change the domestic payable account for the foreign payable account

In our previous meetings, we agreed not to create two separate domestic VAT (for goods and services) because it was not possible to automate when each of those should be used (the res.config allows to configure only one default VAT).

In my view, the MVP could cover just the basic scenarios. For the moment and for having a departure point I've added mapping for sales only and for goods only, it seems clear to the end user as on the invoice line the tax reads clearly "IVA 0% (Exportación BIenes)"

Reply:
@jco-odoo it is done in [this commit](https://github.com/odoo/odoo/pull/92472/commits/c2e01cb8875c5e78637f3b8783631227d07c11d9), please note I've changed the tax sequences for vat sales to multiples of 10 because I needed to put in the middle the newly created taxes (it used to be common practise in previous versions). If you agree I would like to do the same for purchase vat sales.